src/main/java/org/jabref/gui/Main.css

```diff
@@ -308,21 +526,21 @@
 
 .scroll-bar:horizontal .track,
 .scroll-bar:vertical .track {
-    -fx-background-color: derive(#868786, 20%);
+    /*-fx-background-color: derive(-fx-outer-border, 10%);*/
     -fx-opacity: 0.2;
     -fx-background-radius: 0em;
 }
 
 .scroll-bar:horizontal .thumb,
 .scroll-bar:vertical .thumb {
-    -fx-background-color: derive(#757575, 60%);
+    -fx-background-color: -fx-outer-border;
```

Comment:
Introduce a new color for the scrollbars? I could imagine a theme where the scrollbars pop-out a bit more.